packages/closer/pages/events/[slug]/report.tsx

```
@@ -0,0 +1,330 @@
+import dayjs from 'dayjs';
+import { NextApiRequest, NextPageContext } from 'next';
+import { useTranslations } from 'next-intl';
+
+import { useAuth } from '../../../contexts/auth';
+import { usePlatform } from '../../../contexts/platform';
+import { useConfig } from '../../../hooks/useConfig';
+import PageNotAllowed from '../../401';
+import PageNotFound from '../../not-found';
+import api from '../../../utils/api';
+import { parseMessageFromError } from '../../../utils/common';
+import { formatThousands } from '../../../utils/dashboard.helpers';
```

ESLint

[error] 'formatThousands' is defined but never used.

(unused-imports/no-unused-imports)